KerbalStuff/blueprints/api.py

```diff
@@ -103,10 +103,19 @@ def version_info(mod: Mod, version: ModVersion) -> Dict[str, str]:
         "download_path": url_for('mods.download', mod_id=mod.id,
                                  mod_name=mod.name,
                                  version=version.friendly_version),
-        "changelog": version.changelog
+        "changelog": version.changelog,
+        "downloads": mod_version_download_count(mod, version),
     }
 
 
+def mod_version_download_count(mod: Mod, version: ModVersion) -> int:
```

Comment:
What about making this a method of `ModVersion`?
Would be available to other places then as well, and also be somewhere you'd expect it.
Also we don't need to filter by `mod.id` first, since the id of mod versions is unique across all mods.

Reply:
Good idea, done.